examples/text_classification/text_datasets.py

```diff
@@ -29,6 +30,8 @@ def download_dbpedia():
 def read_dbpedia(tf, split, shrink=1, char_based=False):
     dataset = []
     f = tf.extractfile('dbpedia_csv/{}.csv'.format(split))
+    if sys.version_info > (3, 0):
+         f = io.TextIOWrapper(f, encoding='utf-8')
```

Comment:
Would you fix the indentation of this line?
```suggestion
        f = io.TextIOWrapper(f, encoding='utf-8')
```